packages/aggregator/src/connections/EntityConnection.js

```diff
@@ -21,11 +21,14 @@ export class EntityConnection extends ApiConnection {
     dataSourceEntityType,
     dataSourceEntityFilter = {}, // TODO: Add support for dataSourceEntityFilter https://github.com/beyondessential/tupaia-backlog/issues/2660
   ) {
-    return this.get(`hierarchy/${hierarchyName}/relatives`, {
-      entities: entityCodes.join(','),
-      descendant_filter: `type:${dataSourceEntityType}`,
-      field: 'code',
-    });
+    return this.post(
+      `hierarchy/${hierarchyName}/relatives`,
+      {
+        filter: `type:${dataSourceEntityType}`,
```

Comment:
Why did `descendant_filter` become `filter`?

Reply:
It looks like it's correct now, and was maybe a bug before?

Reply:
Yup, was actually a bug before, and a pretty nasty one at that! We were always returning all relatives instead of the ones of the correct level!